### DISCUSSION TITLE
Adding webpack to personal bio site

### DISCUSSION THREAD

**Discussion by chiestroud** [nss-evening-cohort-14/evening-client-side#151] [Help Me!] [answered]
Name: Chie Stroud

Have you searched the other help tickets? If not, do that first (Duplicates asks for help will be shamed ceaselessly)
Yes, but I did not find anything related.

Describe your issue:
Below is one of the requirements for personal bio site no.2. 
"Add webpack
1. Modify your project to add in webpack - this will involve you changing the file structure (src/javascripts/main.js) and convert 2. your css to SCSS files
3. You will probably have a TON of linting errors - FIX THEM
Once this branch is complete you should be able to run npm start and have your project look exactly the way it did before"

I have been cloning / forking the webpack template for other projects and I do not know how to add webpack to an existing project.

What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
https://github.com/nss-nightclass-projects/Night-Class-Resources/blob/master/book-2-patterns-and-tools/chapters/webpack-configure.md
https://scotch.io/tutorials/setting-up-webpack-for-any-project
https://imranhsayed.medium.com/setting-up-webpack-for-your-project-97279bbb0385

What have you tried? Tell us everything you have tried.
1. I cloned webpack-template into my project, and it looks like it only made webpack-template directly inside my project. If I tried to access that file, the project name changes from personal-bio-site to webpack-template.. so I removed it.
2. I installed webpack using the link listed above, but I do not believe that I am supposed to do this. And npm start didn't work so I removed this once again.
3. I thought about making a new repo using webpack template, but I don't think I am supposed to do that.

Can you please help me? I am sorry if this was already mentioned at some point in class, but I cannot recall.

**Comment by drteresavasquez** [comment, marked as answer]
Good question! There are a couple ways you could do this one being to just go ahead and create a new project using the template. If that's the easiest way for you. That works for us. Just put a link to the webpack version in your readme and that should be ok.

**Reply by chiestroud** [reply]
Sounds great! I would like to know the other methods, but my brain has been so fried so I think I will go with the easiest method for now :) Thank you so much!

**Reply by drteresavasquez** [reply]
Remind me and I'll show you the other way you can do it as well.

**Reply by chiestroud** [reply]
Thank you! I am looking forward to learning the new way!

**Comment by GonzalesMatthew** [comment]
I'm going to use the clone method too. The things I found looked doable, but go into a lot of options I'm kind of afraid will lead to a lot of troubleshooting if I get them wrong. 👀
